vllm/core/scheduler.py

```diff
@@ -699,6 +700,7 @@ def _schedule_prefills(
                 logger.warning(
                     "Input prompt (%d tokens) is too long"
                     " and exceeds limit of %d", num_new_tokens, prompt_limit)
+                self._finished_requests_ids.append(seq_group.request_id)
```

Comment:
I believe this can be skipped because `finished_requests_ids` is to release the mamba cache of the already-scheduled requests while the ignored requests here are never scheduled.

Reply:
I agree.